includes/emails/class-edd-email-tags.php

```diff
@@ -474,8 +475,11 @@ function edd_email_tag_download_list( $payment_id ) {
 			}
 
 
-			if ( '' != edd_get_product_notes( $item['id'] ) ) {
-				$download_list .= ' &mdash; <small>' . edd_get_product_notes( $item['id'] ) . '</small>';
+			if ( ! array_key_exists( $item['id'], $needs_notes ) ) {
```

Comment:
The output is totally fine and this is working, but I guess I am looking at this wanting to assign `edd_get_product_notes( $item['id'] )` to a variable and use that as part of the conditional check (in addition to what's already here), instead of building the array for everything in the cart.